librec/src/test/java/librec/util/GaussianTest.java

```diff
@@ -0,0 +1,96 @@
+package librec.util;
+
+import librec.util.Gaussian;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+public class GaussianTest {
+
+  @Rule public ExpectedException thrown = ExpectedException.none();
+
+  /* testedClasses: Gaussian */
+
+  /*
+   * Test generated by Diffblue Deeptest.
+   * This test case covers:
+   *  - conditional line 55 branch to line 56
+   */
+
+  @Test
+  public void cdfInputNegativeOutputZero() {
```

Comment:
In order to get a non- return value from the Gaussian.cdf() tests we'll need to find an input between -8.0 and 8. Is there anything from Deeptest that demonstrates this?